treewide: remove vrthra, cpages and lovek323 from meta.maintainers

AndersonTorres opened the pull request:
## Description of changes

Since they are not active anymore.

https://github.com/NixOS/nixpkgs/pull/310759

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

AndersonTorres commented:
@vrthra gentle ping

thiagokokada commented on pkgs/development/interpreters/guile/2.0.nix:
```diff
@@ -152,7 +152,7 @@ builder rec {
         processing.
       '';
     license = licenses.lgpl3Plus;
-    maintainers = with maintainers; [ ludo lovek323 vrthra ];
```
`lovek323` removed by mistake?

thiagokokada commented on pkgs/development/interpreters/guile/2.2.nix:
```diff
@@ -142,7 +142,7 @@ builder rec {
       foreign function call interface, and powerful string processing.
     '';
     license = licenses.lgpl3Plus;
-    maintainers = with maintainers; [ ludo lovek323 vrthra ];
```
Here too.

thiagokokada commented on pkgs/development/libraries/tk/generic.nix:
```diff
@@ -69,6 +69,6 @@ tcl.mkTclDerivation {
     homepage = "https://www.tcl.tk/";
     license = licenses.tcltk;
     platforms = platforms.all;
-    maintainers = with maintainers; [ lovek323 vrthra ];
```
Here too.

bjornfor commented:
> treewide: remove vrthra from meta.maintainers 

Please explain in the commit message why.

AndersonTorres committed:
treewide: remove vrthra from meta.maintainers [all orphans]

Tracking issue: https://github.com/NixOS/nixpkgs/issues/290642

Inactive since 2017

AndersonTorres committed:
tk: remove vrthra and lovek323 from meta.maintainers [all orphans]

Tracking issue: https://github.com/NixOS/nixpkgs/issues/290642

Inactive since at least 2017 and 2022 respectively.

AndersonTorres committed:
treewide: remove vrthra and lovek323 from meta.maintainers [no orphans]

Tracking issue: https://github.com/NixOS/nixpkgs/issues/290642

Inactive since at least 2017 and 2022 respectively.

AndersonTorres committed:
treewide: remove vrthra from meta.maintainers [no orphans]

Tracking issue: https://github.com/NixOS/nixpkgs/issues/290642

Inactive since at least 2017.

AndersonTorres commented:
> > treewide: remove vrthra from meta.maintainers
> 
> Please explain in the commit message why.

Done

AndersonTorres committed:
treewide: remove cpages from meta.maintainers [all orphans]

Tracking issue: https://github.com/NixOS/nixpkgs/issues/290642

Inactive since 2020

AndersonTorres committed:
treewide: remove cpages from meta.maintainers [no orphans]

Tracking issue: https://github.com/NixOS/nixpkgs/issues/290642

Inactive since 2020

superherointj reviewed (approved):
If they ever want to come back, they can always re-add themselves.

AndersonTorres commented:
P.S.: some packages still refer to them. I  am reworking these packages in other PRs, some of them into Staging.
